src/Microsoft.AspNetCore.JsonPatch/Internal/DictionaryAdapterOfTU.cs

```diff
@@ -126,6 +127,47 @@ public class DictionaryAdapter<TKey, TValue> : IAdapter
             return true;
         }
 
+        public bool TryTest(
+            object target,
+            string segment,
+            IContractResolver contractResolver,
+            object value,
+            out string errorMessage)
+        {
+            var contract = (JsonDictionaryContract)contractResolver.ResolveContract(target.GetType());
+            var key = contract.DictionaryKeyResolver(segment);
+            var dictionary = (IDictionary<TKey, TValue>)target;
+
+            if (!TryConvertKey(key, out var convertedKey, out errorMessage))
+            {
+                return false;
+            }
+
+            // As per JsonPatch spec, the target location must exist for remove to be successful
```

Comment:
???

Reply:
Per my other comment about what's allowed, I think this is correct, no?

Reply:
I need to change it to say "test" instead of "remove". My bad

Reply:
Ah didn't realize this was a about a copy pasta typo... I think the comment is otherwise correct.